mastercraft.cmd

```diff
@@ -2071,6 +2139,7 @@ lack.material:
 purchase.material:
      var purchaselabel purchase.material
      action var need.coin 1 when ^The attendant shrugs and says, "Ugh, you don't have enough
+     #" ### This endquote fixes highlighting problems from the above line

```

Comment:
This just fixes syntax highlighting. BEcause the action above doesn't have an end quote it was highlighting the rest of my code as quoted lol